### DISCUSSION TITLE
Smart Renaming Rules - examples needed.

### DISCUSSION THREAD

**Discussion by tytrate** [aclap-dev/video-downloadhelper#329] [Help and Q&A]
### Your question

Please give some example of how to use Smart Renaming.  I need help understanding what the program is asking for. My goal is to have it produce filenames that don’t have special characters.  I tried this and it failed to make a change. 
![image](https://github.com/aclap-dev/video-downloadhelper/assets/43082324/0e5b8f2b-471d-44f8-8264-557faf58f877)

### Addon & CoApp version and details

version: 8.2.2.8a1
channel: beta
build date: 2024-03-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 04e6283f3f1e4949
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.14",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\tytra",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

**Comment by Mattmjs** [comment]
I don't know a lot about it, I'm a user like you. But I found an example https://rumble.com/v4lexzc-beyond-ravens-with-janine-mar-26.html
I can't get to the file dialog to remove the special characters, there are two at the beginning of the filename.
It just fails right away.
